xgboost_synthetic/README.md

```diff
@@ -1,9 +1,17 @@
 # xgboost-synthetic
-Kubeflow fairing, pipelines demo using synthetic data
+Kubeflow fairing, pipelines demo using synthetic data. This notebook `build-train-deploy.ipynb` can be executed using one of the stock notebook images launched through Kubeflow UI.
 
-1. Launch a notebook
+1. Follow the [Set up your notebook](https://www.kubeflow.org/docs/notebooks/setup/) guide to get started with Jupyter notebooks on Kubeflow
 
-   ```
-   kubectl apply -f notebook.xgboost-synthetic.yaml
-   ```
-1. Attach an extra data volume named 
+1. Open a notebook and run
+    ```
+    !git clone https://github.com/kubeflow/examples.git
+    ```
+
+1. In the directory `xgboost_synthetic`, open the notebook `build-train-deploy.ipynb`
+
+1. Set the environment variable `GOOGLE_APPLICATION_CREDENTIALS` by the command
```

Comment:
Why is this needed? Isn't GOOGLE_APPLICATION_CREDENTIALS already set if they select add GCP credential in the notebook spawner UI?

Reply:
The section "Configurations" doesn't work when I launch a new notebook server using Kubeflow UI.
Another question is that there is no section to configure secrets or environment variables in the [Set up your notebook](https://www.kubeflow.org/docs/notebooks/setup/) page.